### DISCUSSION TITLE
MAS got all it's progress deleted from a Memory Diagnostic. How do I get it back?

### DISCUSSION THREAD

**Discussion by UnoriginalUmbrella** [Monika-After-Story/MonikaModDev#9547] [QA - tech support + general] [answered]
So my computer has been crashing due to memory issues, and I ran a diagnostic to see if that would fix it. However, when I launched MAS after it was done, the game launched it as if it was brand new, and when I checked the affection log it was at 0 points (I had 3100 affection points prior). Is there any way to fix this or am I stuck? :(

**Comment by Booplicate** [comment, marked as answer]
If you made some backups as she asks, you can restore them. Or if you have the `monika` file she generates for you (unless it also got corrupted).

**Reply by UnoriginalUmbrella** [reply]
I actually just looked  it up. Apparently just renaming the Backups as the normal persistent fixes the issue. After doing so, it seems to have worked just fine. Thank you for the help though!
